diagnostic.md

```diff
@@ -5,20 +5,26 @@ Record your responses inside the fenced code blocks below each question.
 1.  Give an example of a visual hierarchy that could be modeled with components.
 
     ```md
-    <!-- your response here -->
+    Nested components
```

Comment:
you are correct.  now what is an example of of UI element that you would use a nested component for?  Common examples would be; a list w/ list-items, a navbar w/ buttons, a sidebar for
a blog w/ comments... etc.